server/spec/content_access_spec.rb

```diff
@@ -883,4 +883,25 @@ def regenerate_cert_test(&updater)
     end
   end
 
+  it 'should be disabled for owner in SCA mode' do
+    consumer = @user.register(random_string('testsystem'), :system, nil,
+               {'system.certificate_version' => '3.1'}, nil, nil, [], [], nil)
+
+    # System purpose status
+    status = @cp.get_purpose_compliance(consumer['uuid'])
+    status['status'].should == 'disabled'
+
+    # compliance status
+    status = @cp.get_compliance(consumer['uuid'])
+    status['status'].should == 'disabled'
+  end
+
+  it "should not allow auto-attach for consumer with activation keys" do
```

Comment:
This test is wrong, so let's remove it. The scenario it describes is opposite of what we want, meaning registration should _never_ fail in SCA mode, regardless of if activation keys are used, or if they have auto-attach set to true or not.
The reason the test passes is unrelated: it seems to be that you accidentally set the value 'true' as the owner key on the register() method.

Reply:
Done.